content/homepages/success-old.html

```diff
@@ -1,5 +1,5 @@
 ---
-slug: success
+slug: success-old
```

Comment:
Should just delete these files, to avoid the old pages being listed and indexed for SEO